source/module_base/global_variable.h

```diff
@@ -231,6 +231,8 @@ extern bool
     deepks_scf; //(need libnpy and libtorch) if set 1, a trained model would be needed to cal V_delta and F_delta
 extern bool deepks_bandgap; // for bandgap label. QO added 2021-12-15
 
+extern bool deepks_equiv; //whether to use equviariant version of DeePKS
```

Comment:
I would suggest not adding new GlobalV variables.

Reply:
Oh then inform me what is the standard practice of using an input variable in the program now? Since if I recalled correctly last time you also suggested not using INPUT directly.